src/__tests__/MissionItem.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import MissionItem from '../Components/MyMissions/MissionItem';
+import '@testing-library/jest-dom';
+import store from '../redux/configureStore';
+
+const testData = {
+  id: '9D1B7E0',
+  name: 'Thaicom',
+  description: 'Thaicom is the name of a series of communications satellites operated from Thailand, and also the name of Thaicom Public Company Limited, which is the company that owns and operates the Thaicom satellite fleet and other telecommunication businesses in Thailand and throughout the Asia-Pacific region. The satellite projects were named Thaicom by the King of Thailand, His Majesty the King Bhumibol Adulyadej, as a symbol of the linkage between Thailand and modern communications technology.',
+  joined: false,
+};
+
+describe('Testing page deployment', () => {
+  test('Mission components matches snapshot', () => {
+    const tree = render(
+      <Provider store={store}>
+        <MissionItem props={testData} />
+      </Provider>,
```

Comment:
- [ ] **[OPTIONAL]:** You did a good job working on your tests! 💯 However, the Redux official docs suggest an approach to handle these multiple calls to the `Provider` on each test and best practices regarding the use of the same store instance for each one of them. If you're curious, take a look at [this article](https://redux.js.org/usage/writing-tests#setting-up-a-reusable-test-render-function). 🤓 🔧